scripts/globals/spells/trust/mumor.lua

```diff
@@ -2,19 +2,78 @@
 -- Trust: Mumor
 -----------------------------------
 require("scripts/globals/trust")
------------------------------------
+-----------------------------------------
 local spellObject = {}
 
+local jobs1 =
+{
+    xi.job.WHM,
+    xi.job.RDM,
+    xi.job.SCH,
+    xi.job.PLD,
+}
+
 spellObject.onMagicCastingCheck = function(caster, target, spell)
-    return xi.trust.canCast(caster, spell, 1015)
+    return xi.trust.canCast(caster, spell)
 end
```

Comment:
Can multiple versions of mumor be cast at the same time on retail?

Reply:
Mumor and Mumor II

Reply:
Shantotto and Shantotto ii can't be cast at the same time on retail if I remember correctly, is that not the case here?

Reply:
Ah, so the spell ID is defined in this case to support the restrictions applied in the trust.lua global. I'll add the spell ID back in.